content/soarian/r4/clinical/care-provision/careteam.md

```diff
@@ -0,0 +1,135 @@
+
+The search results include the following fields if they contain values:
+
+* [Care Team id](https://hl7.org/fhir/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status](https://hl7.org/fhir/careteam-definitions.html#CareTeam.status){:target="_blank"}
+* [Category (Longitudinal or Encounter)](https://hl7.org/fhir/careteam-definitions.html#CareTeam.category){:target="_blank"}
+* [Subject (Patient only](https://hl7.org/fhir/careteam-definitions.html#CareTeam.subject){:target="_blank"}
+* [Encounter (Encounter Category only)](https://hl7.org/fhir/careteam-definitions.html#CareTeam.encounter){:target="_blank"}
+* [Participant](https://hl7.org/fhir/careteam-definitions.html#CareTeam.participant){:target="_blank"}
+  * [Role](https://hl7.org/fhir/careteam-definitions.html#CareTeam.participant.role){:target="_blank"}
+  * [Member](https://hl7.org/fhir/careteam-definitions.html#CareTeam.participant.member){:target="_blank"} (references to RelatedPerson are contained)
+    * [RelatedPerson patient](https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.patient){:target="_blank"}
+    * [RelatedPerson relationship](https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.relationship){:target="_blank"} (text only)
+    * [RelatedPerson name](https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.name){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:soarian_care_team, :r4) %>
+
+## Search
+
+Search for CareTeam resources that meet the specified query parameters:
+
+    GET /CareTeam?:parameters
+
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true, system: true)%>
+
+### Parameters
+
+ Name          | Required?                                    | Type                                                       | Description
+---------------|----------------------------------------------|------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------
+ `_id`         | This, or `patient`                           | [`token`]											       | The logical resource ID associated with the resource.
+ `patient`	   | This, or `_id`								  | [`reference`]											   | The patient who has the care team.
+ `encounter`   | No 										  | [`reference`]											   | The encounter associated with the care team
+ `status`      | No 										  | [`token`]												   | The status of the care team. May be a list separated by commas.
+ `_revinclude` | No 										  | [`_revinclude`]											   | A request to include any Provenance resource in the bundle that refers to an CareTeam resource in the search results. Only supported with Provenance. 
+
```

Comment:
Please change "an CareTeam resource" to "a CareTeam resource". (an to a)

PRODOC-160857